docs/sources/get-started/architecture.md

```diff
@@ -20,36 +20,52 @@ index and in stored chunks.
+  |                       |                    |                        |
+  -----------------------------------------------------------------------
+  |                     #nonIndexedLabels (uvarint)                     |
+  -----------------------------------------------------------------------
+  |      len(label-1) (uvarint)     |          label-1 (bytes)          |
+  -----------------------------------------------------------------------
+  |      len(label-2) (uvarint)     |          label-2 (bytes)          |
+  -----------------------------------------------------------------------
+  |      len(label-n) (uvarint)     |          label-n (bytes)          |
+  -----------------------------------------------------------------------
+  |                     checksum(from #nonIndexedLabels)                |
+  -----------------------------------------------------------------------
+  |           block-1 bytes         |           checksum (4b)           |
+  -----------------------------------------------------------------------
+  |           block-2 bytes         |           checksum (4b)           |
+  -----------------------------------------------------------------------
+  |           block-n bytes         |           checksum (4b)           |
+  -----------------------------------------------------------------------
+  |                          #blocks (uvarint)                          |
+  -----------------------------------------------------------------------
+  | #entries(uvarint) | mint, maxt (varint) | offset, len (uvarint)     |
+  -----------------------------------------------------------------------
+  | #entries(uvarint) | mint, maxt (varint) | offset, len (uvarint)     |
+  -----------------------------------------------------------------------
+  | #entries(uvarint) | mint, maxt (varint) | offset, len (uvarint)     |
+  -----------------------------------------------------------------------
+  | #entries(uvarint) | mint, maxt (varint) |  offset, len (uvarint)    |
+  -----------------------------------------------------------------------
+  |                        checksum(from #blocks)                       |
+  -----------------------------------------------------------------------
+  | #nonIndexedLabels len (uvarint) | #nonIndexedLabels offset (uvarint)|
+  -----------------------------------------------------------------------
+  |     #blocks len (uvarint)       |     #blocks offset (uvarint)      |
+  -----------------------------------------------------------------------
 ```
 
 `mint` and `maxt` describe the minimum and maximum Unix nanosecond timestamp,
 respectively.
 
+The `nonIndexedLabels` section stores non-repeated strings. It is used to store label names and label values from
+[non-indexed labels]({{< relref "./labels/non-indexed-labels" >}}).
+Note that the labels strings and lengths within the `nonIndexedLabels` section are stored compressed using Gzip.
```

Comment:
The compression format is actually configurable using `ingester.chunk-encoding` and defaults to gzip.